clink/app/src/loader/autorun.cpp

```diff
@@ -439,7 +439,7 @@ static void print_help()
 
     puts("Autorun simplifies making modifications to cmd.exe's autorun registry\n"
         "variables. The value of these variables are read and executed by cmd.exe when\n"
-        "it starts. The 'install/uninstall' verbs add/remove the corrent command to run\n"
+        "it starts. The 'install/uninstall' verbs add/remove the current command to run\n"
```

Comment:
The intended word was "correct", instead of "current" (see commit c248c4c, which introduced the typo).